### ISSUE TITLE
Message Deduplication Fails

### ISSUE THREAD

**Issue by chairmantheboring** [oxen-io/session-desktop#1565]
**Describe the bug**
On reimport of a Session ID, message deduplication fails and causes 4 copies (meaning 5 total of the same message) of the message. This is newly happening to me. I've also seen a message have 8 copies (9 total). Some messages duplicate only once (2 total), or duplicate twice (3 total).

**To Reproduce**
1. Import your Session ID to a device.
2. Wait for messages to sync all the way.
3. After all the messages from the last 2 days sync, messages will start to sync again, and these will be the duplicates.

**Screenshots or Logs**
If applicable, add screenshots or log files to help explain your problem.
![image](https://user-images.githubusercontent.com/70743311/114309091-83825380-9ad5-11eb-98e1-2bd9fce4efdf.png)
One-on-one conversation showing duplicates of a message sent by me.
![image](https://user-images.githubusercontent.com/70743311/114309132-b88ea600-9ad5-11eb-84b0-805d8e42725b.png)
Closed group conversation showing duplicates of replies, messages sent by me and messages sent by another member. Both messages with duplicates show up 5 times total.

**Other information (please complete the following information):**
- Device: PC
- OS: Arch Linux
- Session Version or Git commit hash: Session Desktop 1.5.2 (Flatpak)

**Comment by Bilb** [comment]
Hey, was the sender device an android device?

**Comment by chairmantheboring** [comment]
Yeah, I think so.

**Comment by Bilb** [comment]
Probably linked to https://github.com/oxen-io/session-desktop/issues/1590 then. 
A PR is already merged for this and will be on the next release/. I close this one so we track this issue on the 1590 instead
